### PR TITLE
Engagement/jgilmore/create ar receiver transfrom

### PR DESCRIPTION
**Pull request by GilmoreA6**
This PR adds a transform and associated tests and receiver settings

Test Steps:
1. run ./prime test
2. run ./gradlew testIntegration

## Changes
- add hl7 transform for AR
- add integration test for AR transform
- add receiver settings for AR

### Testing
- [x ] Tested locally?
- [x ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?

### CODE REVIEW

File: prime-router/settings/STLTs/AR/AR-phd.yml
Comment (oslynn) [inline prime-router/settings/STLTs/AR/AR-phd.yml:20]: Should we use azure:/ instead of classpath:/?

File: prime-router/settings/STLTs/AR/AR-phd.yml
Comment (oslynn) [inline prime-router/settings/STLTs/AR/AR-phd.yml:6]: Should we put the setting as this or as we did before (i.e ./setting/staging/0170-ar-phd-.....)?

File: prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/AR/AR-receiver-transform.yml
Comment (oslynn) [inline prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/AR/AR-receiver-transform.yml:86]: Love this, thank you.

File: prime-router/settings/STLTs/AR/AR-phd.yml
Comment (GilmoreA6) [reply prime-router/settings/STLTs/AR/AR-phd.yml:20]: We should. the validate-schemas command was broken but Victor put in a PR to fix it so I can change this back to use Azure.

File: prime-router/settings/STLTs/AR/AR-phd.yml
Comment (GilmoreA6) [reply prime-router/settings/STLTs/AR/AR-phd.yml:6]: I'm fine either way. Jamie moved them to this folder when he moved them out of the mappings folder but it doesn't really matter to me.

File: prime-router/settings/STLTs/AR/AR-phd.yml
Comment (victor-chaparro) [inline prime-router/settings/STLTs/AR/AR-phd.yml:48]: First time I see this included 414015000 (Orthopox Viruses). What is that for, when do we need to include it?

File: prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/AR/AR-receiver-transform.yml
Comment (victor-chaparro) [inline prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/AR/AR-receiver-transform.yml:190]: Check if you still need to add the NPI transform after Jessica's changes to the simple report sender transform.

**Review by victor-chaparro — APPROVED** [review]
LGTM, but may need updates after Jessica's changes get merged, maybe wait for that to get merged?

**Pull request closed without merging**
